src/strategies/docker_strategy.rs

```diff
@@ -0,0 +1,95 @@
+use std::process::Command;
+
+use anyhow::{anyhow, bail, Result};
+use log::{info, trace};
+use typed_builder::TypedBuilder;
+
+use crate::commands::build::Credentials;
+
+use super::BuildStrategy;
+
+#[derive(Debug, TypedBuilder)]
+pub struct DockerStrategy {
+    creds: Option<Credentials>,
+}
+
+impl BuildStrategy for DockerStrategy {
+    fn build(&self, image: &str) -> Result<()> {
+        trace!("docker build . -t {image}");
+        let status = Command::new("docker")
+            .arg("build")
```

Comment:
we should use buildx command instead

Reply:
https://docs.docker.com/build/buildkit/

Reply:
I looked into it and it and `docker build` is now the alias to `docker buildx build`.

Reply:
I thought you had to pass DOCKER_BUILDKIT